src/Nri/Ui/Menu/V4.elm

```diff
@@ -955,6 +956,15 @@ viewClickableText title menuConfig clickableTextAttributes attributes =
         ([ ClickableText.custom attributes
          , ClickableText.disabled menuConfig.isDisabled
          , ClickableText.rightIcon (AnimatedIcon.arrowDownUp menuConfig.isOpen)
+         , ClickableText.css
+            [ Css.Global.descendants
+                [ Css.Global.selector "svg"
```

Comment:
I think this will resize the left icon too, if the user adds one, right?

Reply:
right! Thanks! Made some changes